### PR TITLE
GH-12067 adjust cypress test for add configuration directly, checkout…

### LINKED ISSUE #12067: Check Cypress Test on CCV2: should be able to add a configuration directly to ...
See http://cpqjenkins.mo.sap.corp:8080/job/CCV2_CPQ_CYPRESS_TESTS/495/testReport/

Failed Test: should be able to add a configuration directly to the cart, navigate from the cart back to the configuration and update it, checkout and order.

Error Message:
Timed out retrying after 100000ms: Expected to find content: '1' within the element: <span.count> but never did.

### PR DESCRIPTION
**Pull request by AlexanderKuschner**
Adjust cypress test for add configuration directly, checkout and order:
identify cart entry correctly; wait for API call during continue to delivery mode in checkout.

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts [hunk 1]
```diff
@@ -420,100 +419,112 @@ context('CPQ Configuration', () => {
...
+        price: '$1,500.00',
+        quantity: '1',
+      },
+      {
+        name: 'LowePro Streetline SL 140',
+        price: '$110.00',
+        quantity: '1',
+      },
+    ];
+    configuration.checkAmountOfBundleItems(cartEntryIndex, 3);
```
Comment (Larisa-Staroverova) [inline projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts]: Could `configuration.checkAmountOfBundleItems(cartEntryIndex, ovBundleInfos.length)` be used instead?

File: projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts [hunk 2]
```diff
@@ -420,100 +419,112 @@ context('CPQ Configuration', () => {
...
+    configurationOverview.checkAttrDisplayed('Camera Body', 'Canon EOS 80D', 0);
+
+    configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_ACC, 1);
+    configurationOverview.clickContinueToCartBtnOnOP();
+
+    configuration.checkAmountOfBundleItems(cartEntryIndex, 4);
+    configuration.verifyCartCount(numberOfCartItems);
+  }
+
+  function checkout() {
```
Comment (Larisa-Staroverova) [inline projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts]: Not sure why we need this method. Why couldn't we call directly both methods?
```
configuration.clickOnProceedToCheckoutBtnInCart();
configuration.checkoutB2B();
```
without rapping them

File: projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts [hunk 3]
```diff
@@ -420,100 +419,112 @@ context('CPQ Configuration', () => {
...
+    configuration.clickOnProceedToCheckoutBtnInCart();
+    configuration.checkoutB2B();
+  }
+
+  function orderHistory() {
```
Comment (Larisa-Staroverova) [inline projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts]: The same here: why we rap these methods. Why couldn't we call directly them?

**Review by Larisa-Staroverova — CHANGES_REQUESTED** [review]
Please, see my comments

File: projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts (hunk 1 above)
Comment (AlexanderKuschner) [reply projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts]: done

File: projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts (hunk 2 above)
Comment (AlexanderKuschner) [reply projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts]: done

File: projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts (hunk 3 above)
Comment (AlexanderKuschner) [reply projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts]: done

**Review by Larisa-Staroverova — APPROVED** [review]

**Pull request merged**